setup.py

```diff
@@ -37,9 +37,7 @@ def get_install_requires():
         'cliff',
         'colorlog',
         'numpy',
-        # TODO(Yanase): Remove the version constraint when the CI error is solved by new versions.
-        # See https://github.com/optuna/optuna/issues/800 for further details.
-        'scipy<1.4.0',
+        'scipy>=1.4.1',
```

Comment:
I believe we can still keep supporting older versions.
```suggestion
        'scipy<1.4.0,>=1.4.1',
```

Reply:
Sorry for my incorrect suggestion. You're absolutely right about `!=`.